frontend/app/actions/CommonActions.scala

```diff
@@ -14,16 +15,23 @@ import play.api.mvc.Results._
 import play.api.mvc._
 import services.{AuthenticationService, TouchpointBackend}
 import utils.GuMemCookie
-import utils.TestUsers.isTestUser
+import utils.TestUsers.{PreSigninTestCookie, isTestUser}
 
 import scala.concurrent.Future
 
-trait CommonActions {
+trait CommonActions extends LazyLogging {
   import OAuthActions._
 
   val AddAbTestingCookiesToResponse = new ActionBuilder[Request] {
     def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]) =
-      block(request).map { _.withCookies(ABTest.cookiesWhichShouldBeDropped(request) ++ AudienceId.cookieWhichShouldBeDropped(request) :_*) }
+      block(request).map { result =>
+        val newABTestCookies = ABTest.cookiesWhichShouldBeDropped(request)
+        if (newABTestCookies.nonEmpty) {
+          val testUser = isTestUser(PreSigninTestCookie, request.cookies)(request).isDefined
+          logger.info(s"dropping-new-ab-test-cookies (path=${request.path} testUser=$testUser) : ${newABTestCookies.map(c => s"${c.name}=${c.value}").mkString(" ")}")
```

Comment:
Yeah, I think so. This will actually be where I measure entry in this funnel - when the AB cookie for `merged-social-registration` is dropped.